### ISSUE TITLE
Feature Request: Add option to scale font

### ISSUE THREAD

**Issue by marhkb** [psifidotos/applet-window-title#20]
AWC has this and I would love to see this feature for applet-window-title
and applet-window-appmenu. For me the font is far too small compared to the font in my window decorations. Please give the us the option to customize that.

**Comment by marhkb** [comment]
Yes I can set it from systemsettings->fonts, but that affects all my other fonts, too - what I don't like. No problem, I will stick with AWC then for the future.

Thank you

**Comment by psifidotos** [comment]
No prob, whatever suits you best
